components/buttongroup/index.css

```diff
@@ -11,7 +11,28 @@ governing permissions and limitations under the License.
 */
 
 .spectrum-ButtonGroup {
-  --spectrum-buttongroup-button-gap-reset: 0;
+  --spectrum-buttongroup-spacing-horizontal: var(--spectrum-spacing-300);
```

Comment:
Should we update the YAML to show the t-shirt sizes and help confirm correct spacing for them?

Reply:
I added variants to show the different between M, L, XL and S. Is this what you had in mind?

Reply:
Yes! But the buttons within the group need to be changed to --sizeS also.

Good call to show m/l/xl together since their spacing is the same.

Reply:
Ah, thank you. Updated.